frontend/src/components/Button.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
```

Comment:
This component is not a general Button class so consider renaming it.

Reply:
I tried making it a Button class. but the tag "button" has features that are not useful in this context. So if we stick with the anchor tag, what do you suggest the name of the component be?